packages/boxel-ui/addon/components/add-button/usage.gts

```diff
@@ -1,11 +1,14 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import FreestyleUsage from 'ember-freestyle/components/freestyle/usage';
 import BoxelAddButton from './index';
 import { action } from '@ember/object';
 import { on } from '@ember/modifier';
 import { fn } from '@ember/helper';
 
 export default class AddButtonUsage extends Component {
+  @tracked variant?: 'full-width';
```

Comment:
What's the advantage of adding a property over adding a `full-width` class to the `AddButton` component?

Reply:
I thought there might be more variants in the future, but am willing to change it. What's your suggestion? @jurgenwerk

Reply:
Hm, I thought you are just adding a variant where the button gets stretched to claim the whole available width (i.e `width: 100%`, but actually now I see the 2 variants look very different:

<img width="61" alt="image" src="https://github.com/cardstack/boxel/assets/273660/4f285cb3-579d-49a4-a29e-8ff8c0e36374">
<img width="443" alt="image" src="https://github.com/cardstack/boxel/assets/273660/3d048803-39ef-4ae0-a605-fe21f19b5ffd">

I was confused by the "full width" naming. I get it now